### PR TITLE
Rebased

### PR DESCRIPTION
**Pull request by KPP13**
First pull request with initial configuration of our project. I added small changes in last commit. Everything is fine, code compiles and is the branch is able to merge now.

### CODE REVIEW

**Comment by danone771909** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+os:
+  - linux
+jdk:
+  - oraclejdk8
+language: java
+sudo: false
+script: mvn clean verify
+notifications:
```
Comment (LIttleAncientForestKami) [inline .travis.yml:8]: :+1:

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,153 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# our ignored files
```
Comment (LIttleAncientForestKami) [inline .gitignore:14]: Have you pulled all changes? As I'm quite sure that I specified `intellij` for `gi`. Also, below you have number of entries for `idea` dir.

File: .gitignore [hunk 3]
```diff
@@ -10,3 +10,153 @@
...
+# auto-generated tag files
+tags
+
+
+### Intellij ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff:
+.idea/workspace.xml
```
Comment (LIttleAncientForestKami) [inline .gitignore:40]: If you ignore whole `.idea` then cut things like that please.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,8 @@
 # robot
 Screen scraping project for e-libraries
+
+# contributors
```
Comment (LIttleAncientForestKami) [inline README.md:4]: :+1:

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+    <properties>
+        <maven.compiler.jdk.version>1.8</maven.compiler.jdk.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.version>3.5.1</maven.compiler.version>
+        <jacoco.version>0.7.7.201606060606</jacoco.version>
+        <testng.version>6.9.10</testng.version>
+        <spring.version>4.3.1.RELEASE</spring.version>
+    </properties>
```
Comment (LIttleAncientForestKami) [inline pom.xml:19]: Nice! :heart_decoration:

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.4.0.RELEASE</version>
```
Comment (LIttleAncientForestKami) [inline pom.xml:24]: :mag_right: :?:

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven.compiler.version}</version>
+                <configuration>
+                    <source>${maven.compiler.jdk.version}</source>
+                    <target>${maven.compiler.jdk.version}</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19</version>
```
Comment (LIttleAncientForestKami) [inline pom.xml:59]: :mag_right:

File: pom.xml [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+                <configuration>
+                    <!-- Sets the VM argument line used when unit tests are run. -->
+                    <argLine>${argLine}</argLine>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.7.7.201606060606</version>
```
Comment (LIttleAncientForestKami) [inline pom.xml:68]: And again version directly in POM... :mag_right:

File: src/test/java/TestTest.java [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertTrue;
```
Comment (LIttleAncientForestKami) [inline src/test/java/TestTest.java:3]: Why assertion for JUnit if you cut it? Wouldn't `assertj` be better? :open_mouth:

Comment (danone771909) [reply src/test/java/TestTest.java:3]: You're right. It will be corrected :+1:

File: pom.xml (hunk 8 above)
Comment (danone771909) [reply pom.xml:68]: We'll change it and add version in `properties` :memo:

File: src/test/java/TestTest.java (hunk 9 above)
Comment (KPP13) [inline src/test/java/TestTest.java]: It is Intellij's default auto import, that's why...

**Pull request merged**